src/commands/common/pipeline-renderer.ts

```diff
@@ -73,18 +56,48 @@ export class PipelineRenderer {
         }),
     );
 
+    return table.toString();
+  }
+
+  public async confirmPipeline(
+    pipeline: Pipeline,
+    autoApprove?: boolean,
+  ): Promise<void> {
+    if (autoApprove) {
+      return;
+    }
+    Inputs.assertInteractiveShell('Use the flag \'--auto-approve\' to skip interactive approval');
+    const tableStr = this.pipelineToTableOutput(pipeline);
+    console.log(tableStr);
+    const shouldContinue = await Inputs.promptForContinuation('Do you want to apply the above changes?');
+    if (!shouldContinue) {
+      Deno.exit(0);
+    }
+
+    console.log(ansiEscapes.eraseLines(tableStr.split('\n').length + 2) + ansiEscapes.cursorMove(0, -1));
```

Comment:
Does this work correctly if the terminal is using line wrapping because the text does not neatly fit.

Reply:
Yeah this does not work right. Looking for a fix.